_ml-commons-plugin/api.md

```diff
@@ -156,13 +156,22 @@ The response contains the following model information:
 
 ## Registering a model
 
-Before you register a model, you must [register a model group]({{site.url}}{{site.baseurl}}/ml-commons-plugin/model-access-control#registering-a-model-group) for the model.
-{: .important} 
+All versions of a particular model are held in a model group. You can [register a model group]({{site.url}}{{site.baseurl}}/ml-commons-plugin/model-access-control#registering-a-model-group) first, before registering a model to the model group. Each model group name must be globally unique in the cluster. 
```

Comment:
```suggestion
All versions of a particular model are held in a model group. You can either [register a model group]({{site.url}}{{site.baseurl}}/ml-commons-plugin/model-access-control#registering-a-model-group) before registering a model to the group, or register a first version of a model, thereby creating the group. Each model group name must be globally unique in the cluster. 
```

Reply:
Something like that. It's a bit of a tongue twister.

Reply:
@natebower Could you help rephrase if needed?

Reply:
@kolchfa-aws I edited @hdhalter's suggestion.